src/pages/Home/utils/handleUserData.ts

```diff
@@ -0,0 +1,55 @@
+import { ref } from 'vue';
+import { MemberInformation, UserType } from '../mock';
+
+export const handleUserData = () => {
+  const getExpirationDate = (expiration: string) => {
+    const expirationDate = ref<Date>(new Date('01/01/1999'));
+    const isMembershipActive = ref<boolean>(false);
+
+    const _currentDate = new Date();
+
+    const _expirationString = expiration;
+
+    if (_expirationString) {
+      expirationDate.value = new Date(_expirationString);
+    }
+
+    isMembershipActive.value =
+      _currentDate.getTime() < expirationDate.value.getTime();
+
+    return { expirationDate, isMembershipActive };
+  };
+
+  const getUserName = (memberInformation: MemberInformation) => {
+    const firstName = ref<string>('');
+    const lastName = ref<string>('');
+
+    const { first_names } = memberInformation;
+    const { last_names } = memberInformation;
+
+    firstName.value = first_names.split(' ')[0];
+    lastName.value = last_names.split(' ')[0];
+
+    return {
+      firstName,
+      lastName,
+    };
+  };
+
+  const getUserMemberCode = (username: string) => {
+    const userCode = ref<string>(username);
+
+    return { userCode };
+  };
```

Comment:
I don't get the purpose of this one, please explain

Reply:
Yes, it's setting the state, I did it that way just to avoid do it on parseUserData function (please notice that parseUserData is just calling functions and returning the results directly), but can be set on parseUserData if you think it's more readable

Reply:
Yes, please use parseUserData instead

Reply:
also notice the comment

> it would change from 'VP000001' to "Victor Colmenarez", is that the intended behavior?